ajax/libs/slabText/package.json

```diff
@@ -14,5 +14,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/freqdec/slabText"
-  }
+  },
+  "licenses": [
+    "MIT",
+    "GPL-2.0"
```

Comment:
I didn't see GPL-2.0 in the reference, would you please confirm? Thanks.

Reply:
Here @extend1994 comment with confirmation
https://github.com/cdnjs/cdnjs/pull/11847/files/07ba1fce38e852c4365a8c38aade4f7e36ae48a3#diff-597e99317c4312b5064e96d95963b8d3

Reply:
The link doesn't work, any other reference?

Reply:
Never mind, I found it.